gcloud/pubsub/_gax.py

```diff
@@ -180,7 +181,279 @@ def topic_list_subscriptions(self, topic_path):
         return subs, response.next_page_token
 
 
+class _SubscriberAPI(object):
+    """Helper mapping subscriber-related APIs.
+
+    :type gax_api: :class:`google.pubsub.v1.publisher_api.SubscriberApi`
+    :param gax_api: API object used to make GAX requests.
+    """
+    def __init__(self, gax_api):
+        self._gax_api = gax_api
+
+    def list_subscriptions(self, project):
+        """List subscriptions for the project associated with this API.
+
+        See:
+        https://cloud.google.com/pubsub/reference/rest/v1/projects.subscriptions/list
+
+        :type project: string
+        :param project: project ID
+
+        :rtype: tuple, (list, str)
+        :returns: list of ``Subscription`` resource dicts, plus a
+                "next page token" string:  if not None, indicates that
+                more topics can be retrieved with another call (pass that
+                value as ``page_token``).
+        """
+        options = CallOptions(is_page_streaming=False)
+        path = 'projects/%s' % (project,)
+        response = self._gax_api.list_subscriptions(path, options)
```

Comment:
`google.pubsub.v1.subscriber_api.SubscriberApi._PROJECT_PATH_TEMPLATE` is a (private) `google.gax.path_template.PathTemplate` instance for this pattern